src/Practitioner/Settings/index.js

```diff
@@ -1,215 +1,289 @@
+import React from 'react';
 import { makeStyles } from '@material-ui/core/styles';
-import useStores from '../../Basics/UseStores';
 import Colors from '../../Basics/Colors';
 import { observer } from 'mobx-react';
 import Language from '../../Components/Shared/LanguageQuestion';
 import { useTranslation } from 'react-i18next';
-import { Box, ButtonBase, Typography } from '@material-ui/core';
+import { ButtonBase, Typography } from '@material-ui/core';
 import GlobeIcon from '@material-ui/icons/Public';
 import PasswordIcon from '@material-ui/icons/Lock';
 import LogoutIcon from '@material-ui/icons/ExitToApp';
 import PasswordReset from '../../Components/PasswordUpdate';
-import Documents from './Documents'
+import Documents from './Documents';
 import PatientIcon from '@material-ui/icons/Accessibility';
 import DocIcon from '@material-ui/icons/Description';
 import ReportProblem from '@material-ui/icons/ReportProblem';
-import PatientInformation from '../../Patient/Information'
-import useLogout from '../../Basics/Logout'
-import Profile from './Profile';
+import PatientInformation from '../../Patient/Information';
+import useLogout from '../../Basics/Logout';
+import useWindowSize from '../../Hooks/Resize';
+import { Switch, Route, Link, useLocation } from 'react-router-dom';
+import { PageLabel } from '../../Components/Shared/PageLabel';
 
 const useStyles = makeStyles({
-    image: {
-        height: "100px",
-        marginLeft: "auto"
-    },
-    report: {
-        display: "flex",
-        width: "100%",
-        border: "2px solid lightgray"
+  image: {
+    height: '100px',
+    marginLeft: 'auto',
+  },
+  report: {
+    display: 'flex',
+    width: '100%',
+    border: '2px solid lightgray',
+  },
+  container: {
+    width: '100%',
+    height: '100%',
+    display: 'flex',
+    marginLeft: '1em',
+  },
+  reportContainer: {
+    width: '50%',
+  },
+  patient: {
+    backgroundColor: 'lightgray',
+  },
+  button: {
+    display: 'block',
+    margin: 'auto',
+    marginTop: '2em',
+  },
+  navigation: {
+    boxSizing: 'border-box',
+    width: '250px',
+    minWidth: '250px',
+    height: '100%',
+    padding: '1em',
+    borderRight: 'solid 1px lightgray',
+  },
+  mobileNav: {
+    boxSizing: 'border-box',
+    width: '100%',
+    height: '100%',
+    padding: '1em',
+  },
 
-    },
-    container: {
-        width: "100%",
-        height: "100%",
-        display: "flex",
-        marginLeft: "1em"
-    },
-    reportContainer: {
-        width: "50%"
-    },
-    patient: {
-        backgroundColor: "lightgray"
-    },
-    button: {
-        display: "block",
-        margin: "auto",
-        marginTop: "2em"
-    },
-    navigation: {
-        boxSizing: "border-box",
-        width: "250px",
-        minWidth: "250px",
-        height: "100%",
-        paddingTop: "1em",
-        borderRight: "solid 1px lightgray"
-    },
-    navItemContainer: {
-        listStyle: "none",
-        height: "2.5em"
-    },
-    navItem: {
```

Comment:
Since this css is quite complicated, maybe its better to create a new file to clean this up a bit

Reply:
ie move NavItem into its own component

Reply:
CSS and NavItem component moved into NavItem.js